pkg/util/render.go

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Can we cut and actual package name for this? Maybe templater, you pick a name. Util pkg name drives me nuts.